src/librustc/traits/structural_impls.rs

```diff
@@ -175,7 +175,7 @@ impl<'a, 'tcx> Lift<'tcx> for traits::SelectionError<'a> {
             super::ConstEvalFailure(ref err) => tcx.lift(&**err).map(|err| super::ConstEvalFailure(
                 err.into(),
             )),
-            super::Overflow => bug!(), // FIXME: ape ConstEvalFailure?
```

Comment:
Hmm, I'm .. not entirely sure why this was `bug!` to begin with. It seems like it could just be `Some(super::Overflow)`, instead?